lib/api/controllers/documentController.js

```diff
@@ -926,4 +933,4 @@ function hasMultiTargets (str) {
   return [',', '*', '+'].some(chr => str.includes(chr)) || str === '_all';
 }
 
-module.exports = DocumentController;
+module.exports = { DocumentController, actions };
```

Comment:
Instead of modifying the export and declaring the actions separately, you could declare a static property on the DocumentController containing your action <-> event mappings